src/main/java/uk/gov/hmcts/sptribs/document/DocumentUtil.java

```diff
@@ -55,6 +55,10 @@ public static List<String> validateCaseworkerCICDocumentFormat(List<ListValue<Ca
         return errors;
     }
 
+    public static List<ListValue<CaseworkerCICDocument>> cleanDocumentList(List<ListValue<CaseworkerCICDocument>> documentList) {
```

Comment:
As per this approach the dev needs to clean the document list in every places. So, there is an additional challenge to keep the consistency across the board even when we introduce new documents.
My suggestion would be to add null checks in the respective places rather than calling an additional method for cleaning up the documents in every place.
Example: For the current issue, the document list null check can be added to the methods updateCategoryToCaseworkerDocument and updateCategoryToDocument instead of using this method